scripts/validate_cache.js

```diff
@@ -0,0 +1,26 @@
+import { Ajv } from 'ajv'
+import addFormats from 'ajv-formats'
+import { readdirSync, readFileSync } from 'fs'
+
+const ajv = new Ajv()
+addFormats(ajv)
+
+const schemaPath = './schema/cache-v1.json'
```

Comment:
Why do we have `cache-v2.json` when we validate against version 1?

Reply:
Who executes this file?

Reply:
So far, nobody. The `cache-v2.json` schema corresponds to the new schema introduced in https://github.com/enso-org/enso/pull/10224. This is just a preparation for when https://github.com/enso-org/enso/pull/10224 gets integrated (I want to integrate only the new data schema, not the rest of the changes in that PR).